connectors/github-pr/src/main/java/com/vmware/connectors/github/pr/GithubPrController.java

```diff
@@ -506,20 +510,20 @@ private String getActionUrl(String routingPrefix, PullRequestId id, String actio
     public Single<ResponseEntity<String>> merge(
             @RequestHeader(AUTH_HEADER) String auth,
             @RequestHeader(BASE_URL_HEADER) String baseUrl,
-            PullRequestId id,
+            PullRequestId pullRequestId,
             @RequestParam("sha") String sha
     ) {
         logger.trace(
-                "merge called: baseUrl={}, owner={}, repo={}, number={}",
+                "merge called: baseUrl={}, pull request id={}",
```

Comment:
Fixed error. In the log we are trying to print 4 fields but we are passing only 2 parameters.

Reply:
At some point I refactored it from the 3 separate fields to the id, but it looks like I had missed a log string literal, thanks!